main.go

```diff
@@ -0,0 +1,173 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/bitrise-io/go-utils/command"
+	"github.com/bitrise-io/go-utils/fileutil"
+	"github.com/bitrise-io/go-utils/log"
+	"github.com/bitrise-tools/go-steputils/stepconf"
+	"github.com/bitrise-tools/go-steputils/tools"
+)
+
+// Config ...
+type Config struct {
+	SSHRsaPrivateKey        stepconf.Secret `env:"ssh_rsa_private_key,required"`
+	SSHKeySavePath          string          `env:"ssh_key_save_path"`
+	IsRemoveOtherIdentities bool            `env:"is_remove_other_identities,required"`
+}
+
+func main() {
+	var cfg Config
+	if err := stepconf.Parse(&cfg); err != nil {
+		failf("Issue with input: %s", err)
+	}
+
+	stepconf.Print(cfg)
+	fmt.Println()
+
+	// Remove SSHRsaPrivateKey from envs
+	if err := os.Unsetenv("ssh_rsa_private_key"); err != nil {
+		failf("* Failed to remove ssh_rsa_private_key")
```

Comment:
avoid this old style logging, we use colors to highlight the log lines, no need for`*` or other chars